processing/src/main/java/org/apache/druid/segment/nested/NestedDataColumnV5.java

```diff
@@ -30,16 +30,24 @@
 import org.apache.druid.segment.data.FrontCodedIntArrayIndexed;
 import org.apache.druid.segment.data.GenericIndexed;
 import org.apache.druid.segment.data.Indexed;
-import org.apache.druid.segment.serde.StandardTypeColumnSerializer;
 
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.util.List;
 
-public class StandardNestedDataColumn<TStringDictionary extends Indexed<ByteBuffer>>
+/**
+ * Nested data column with optimized support for simple arrays. Not actually v5 in the segment since columns are now
+ * serialized using {@link org.apache.druid.segment.serde.NestedCommonFormatColumnPartSerde} instead of the generic
+ * complex type system.
+ *
+ * Not really stored in a segment as V5 since instead of V5 we migrated to {@link NestedCommonFormatColumn} which
+ * specializes physical format based on the types of data encountered during processing, and so versions are now
+ * {@link NestedCommonFormatColumnSerializer#V0} for all associated specializations.
```

Comment:
I like the `NestedCommon` name that you picked for other stuff.  It wouldn't bother me if you named this `NestedCommon` as well, fwiw.  I'm also fine with the current naming, so more stream-of-consciousness thought than a request for change.